docs/EditContext/explainer.md

```diff
@@ -146,7 +149,7 @@ enum EditContextEnterKeyHint {
     "send"
 };
 
-enum EditContextInputPolicy {
+enum EditContextInputPanelPolicy {
```

Comment:
What do we mean by panel here? Besides an on-screen keyboard. Is IME candidate window considered to be an input panel? What about inking panel?

Reply:
I think this is related to virtual keyboard. We should probably change this to EditContextVirtualKeyboardPolicy to match our virtualKeyboardPolicy attribute proposal